.github/workflows/checks.yml

```diff
@@ -82,6 +84,9 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Install All Optional Dependencies (aka. "extras")
```

Comment:
PTB currently only offers action `python-environment` to be called with a list of space-separated extras.
Proposal: Add support for requesting to install "all extras".